contracts/strategies/StrategyGenLevAAVE.sol

```diff
@@ -6,7 +6,7 @@ pragma solidity >=0.6.0 <0.7.0;
 pragma experimental ABIEncoderV2;
 
 // These are the core Yearn libraries
-import {BaseStrategyInitializable} from "../BaseStrategy.sol";
+import {BaseStrategy} from "../BaseStrategy.sol";
```

Comment:
You may want to stick with `BaseStrategyInitializable`, i moved away to reduce bytecode size and switch to the factory pattern for cloning, but it may not be necessary for this slimmer version.